docs/smart-contract-vm.md

```diff
@@ -65,7 +65,7 @@ You should see a message saying "Database created".
 Start by type checking the "names" sample contract.
 
 ```bash
-$ blockstack-core local check sample-programs/names.clar /data/db
+$ blockstack-core local check sample-contracts/names.clar /data/db
```

Comment:
Is `blockstack-core` still the name of the binary to run?

Reply:
Yep -- these commands are also checked in a "demo_tests" job in Circle.